legacy/app/data/post-create/post-editor.directive.js

```diff
@@ -132,8 +132,11 @@ function PostEditorController(
                             attr.value.value = dayjs(attr.value.value).format('YYYY-MM-DD');
                         } else if (attr.default) {
                             try {
-                                attr.value.value = dayjs(new Date(attr.default).format('YYYY-MM-DD'));
-                            } catch (err) {
+                                let defaultValue = dayjs(new Date(attr.default)).format('YYYY-MM-DD');
+                                // Safeguarding against invalid default dates below. We should add validation in the survey setup instead
+                                if (defaultValue !== 'Invalid Date') {
+                                    attr.value.value = defaultValue;
+                                }                            } catch (err) {
```

Comment:
Please check the indentation here, I don't usually look too much into it but this is a bit too much off :)

Reply:
That is awful! Fixed!